src/components/domain/project/projectRegister/step/Step2.tsx

```diff
@@ -11,6 +11,7 @@ import { Input } from '@/components/ui/input';
 import { XCircle } from 'lucide-react';
 import { FieldErrors, useFieldArray, useFormContext } from 'react-hook-form';
 import type { ProjectForm, ProjectMember } from '@/models/projectModels';
+import Planet1 from '../../../../../../public/icon/planet/planet-1.svg';
```

Comment:
svgr 사용 좋습니다! 다만 지금 경로가 너무 지저분한데, 절대 경로로 바꾸거나 `icon/planet` 폴더에 `index.ts` 하나 추가해서 불러오면 좋을 것 같습니다.

Reply:
```suggestion
import Planet1 from '/public/icon/planet/planet-1.svg';
```
nextjs에서 public은 절대경로로 인식 가능하다고 하네요! 위처럼 하면 될 것 같습니다.

근데 public 폴더(정적 파일 저장)에는 ts, tsx 파일을 못사용하는걸로 알고있는데, src 내에 assets를 하나 만들어야할까요?

Reply:
저는 보통 `src/assets`에서 `svg`를 관리하고, `public`에서는 고정 이미지만 저장했습니다. 괜찮으시다면 `src/assets/icons`에 모아두고 `index.ts`에서 뽑아서 사용하는 방향은 어떠신가요?

Reply:
✅ e16243638522dcaaddd89e92be5423b6a979a5b1
위와 같이 수정했습니다